backend/tracim_backend/models/data.py

```diff
@@ -1173,6 +1173,18 @@ def children(self) -> ["Content"]:
 
         return valid_children
 
+    @property
+    def recursive_children(self) -> ["Content"]:
+        """
+        Get all children recursively (including children of children...)
+        """
+        children = []
+        for child in self.children:
+            children.append(child)
+            for sub_child in child.recursive_children:
```

Comment:
Instead of:
```
            for sub_child in child.recursive_children:
                children.append(sub_child)
```

Write it like this:

```
            children.extend(child.get_recursive_children())
```

By the way, it would be more elegant to have a method `get_children(recursivly=True)`

Reply:
done.